deliverables/README.md

```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
+First method is creating a separate CSS file which can be linked to the web page being built. Second method
+is creating a CSS block within the html code, most often being placed in between the opening and closing "head"
+tags. And the third method is insertion of the CSS code right in the html tag itself.
 
 #### Android
 Put your html code in the `android.html` file in this folder.
+Here is the link to my android preview:
+**http://htmlpreview.github.io/?https://github.com/marua7665/2015week1/blob/master/deliverables/android.html**
 
 ---
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**                                                            
+Git is a distributed revision control system. What this means is that a group of people can work on the
+same project without having to share a common network.
+
+**What is SCM?**                                                                  
+SCM is a source control management system that allows for management of changes to documents or other large
+collections of information. SCM can be embedded in other systems to allow for fluid work with edits, making
+collaboration very easy.
+
+**What is a VCS?**                                                                         
+VCS are version control systems that behave similarly to SCM, but often run as stand-alone applications.
+
+**Why is Git useful for a developer?**                                                        
+For a developer, git is very useful because it saves different versions of the project being worked on. 
+Any change in a file is basically one version of that file, which the developer can access. This 
+also makes sharing the workload, as well as editing, very easy.
+
+**Why is Git useful for a team of developers?**                                             
+When a group of developers are working on the same project, fluidity in their collaboration is very 
+important. The fact that each of the developers are working on their own version of the project means that
+they can work on it and bring it back to the central file, with their changes accredited to their names.
```

Comment:
Also changes can be brought into the central repository in a controlled fashion.  So even if one feature was written in Febraury and another in March, the one written in March can be added first, or just specific commits from that feature branch can be cherry-picked into the main branch.